zespolona.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Nov 28 16:39:05 2017
+
+@author: Anna
+"""
+
+class Zespolona:
+    def __init__(self,rz,ur):
+        self.r=rz
+        self.i=ur
+        print(self.r, ',', self.i,'i')
+    def __add__(self,cos):

```

Comment:
Nie używaj nazwy zmiennej "coś". cos to cosinus i ciężko się takie coś czyta ;)

Reply:
zmienione :)